lib/protocol/packets/index.js

```diff
@@ -20,3 +20,4 @@ exports.RowDataPacket = require('./RowDataPacket');
 exports.SSLRequestPacket = require('./SSLRequestPacket');
 exports.StatisticsPacket = require('./StatisticsPacket');
 exports.UseOldPasswordPacket = require('./UseOldPasswordPacket');
+exports.ComInitDbPacket = require('./ComInitDbPacket');
```

Comment:
The contents of this file are in alphabetical order if you can reorder this line to match :+1:

Reply:
Hi,
No, the changes I did were just for FakeServer.js. It was not my intention to expose COM_INIT_DB in the client API as I don't think it is really necessary. 
The situation is this: I was put in charge of maintaining a python application but there were no regression tests written for it. So I came up with the idea of writing a tool to automatically write regression test for it by capturing all mysql interaction but for this I needed a library for the mysql protocol. So I checked this module and realized I could just use the FakeServer. But the python app does a (probably unnecessary) call to mysql_select_db (that issues COM_INIT_DB) and so my tool was failing.

I can add a method init_db in the client API just for the FakeServer to be testable but I think no client code will actually use this and so I would not like to pollute the interface with it. But if you think it is OK, I can do it.

Reply:
How are you even using the FakeServer? We don't publish that file to npm and it is subject to sudden drastic changes because it's private to our use-case. The "mysql2" module actually provides a server implementation that is real and supported. Is that not what you should be using?

The only changes accepted to FakeServer.js are those that are directly related to the functionality in this module; as soon as something in FakeServer.js is not used in a test here, I delete it, and this PR would just end up immediately deleted since there are no tests making use of the functionality in FakeServer.js

Reply:
Hi, I use yarn specifying the URL for the github repo in package.json:
  "mysql": "https://github.com/mysqljs/mysql#v2.15.0"
Yarn gets the test folder.
Sorry for the noise here and thanks for the tip about mysql2.

Reply:
Very interesting. That looks like it just clones our repo. I guess that means from there, you're just requiring the file directly. Hmm, not sure if it's possible we can hide our one-off testing code from that sort of loading, though we don't document using that anywhere. When I first slapped that code together, the "mysql2" module didn't have server code, but now it does. Eventually that would be removed and "mysql2" or another module used and your code would probably end up broken at some point.

This module's code is also MIT licensed, so you're free to copy and paste it into your own repo to use at your liking as well.

Reply:
Yes. Here is the code:
  https://github.com/MayamaTakeshi/mysql_query_collector/blob/master/src/index.js
(sorry, no documentation as this is ongoing work not for public consumption yet)
Thanks.